app/main.py

```diff
@@ -2,0 +1,32 @@
+class Animal:
+
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(prey: object) -> None:
+        if isinstance(prey, Herbivore) and not prey.hidden:
+            prey.health -= 50
+        if prey.health <= 0:
+            prey.health = 0
+            if prey in Animal.alive:
+                Animal.alive.remove(prey)
```

Comment:
Ensure that when an animal is removed from `Animal.alive`, the list is formatted correctly when printed. The task requires that printing `Animal.alive` should display a list of formatted strings representing each animal.